source/postprocess/point_values.cc

```diff
@@ -77,6 +103,10 @@ namespace aspect
                                  current_point_values[p].end());
           Utilities::MPI::sum (v, this->get_mpi_communicator(),
```

Comment:
The comment about `sum` is actually no longer correct -- we now require 8.4, and that version can sum `Vector` arguments, so the workaround is no longer necessary. While you're there, could you fix this as well?